spec/fixtures/composite-action.yml

```
@@ -0,0 +1,40 @@
+runs:
+  using: "composite"
+  steps:
+    - uses: aws-actions/configure-aws-credentials@v1
+      with:
+        # Actions cannot access secrets so pass them in as inputs
+        aws-access-key-id: ${{ inputs.aws-access-key-id }}
+        aws-secret-access-key: ${{ inputs.aws-secret-access-key }}
+        aws-region: eu-west-1
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 14
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 9 but found 8

(indentation)